app/web/features/messages/requests/HostRequestStatusIcon.tsx

```diff
@@ -18,23 +18,21 @@ export default function HostRequestStatusIcon({
   let icon = null;
   let color = null;
 
-  const isRequestPast = dayjs(hostRequest.toDate).isBefore(dayjs().format("L"));
-
   if (s === HostRequestStatus.HOST_REQUEST_STATUS_ACCEPTED) {
     icon = <CheckIcon fontSize="inherit" />;
-    color = isRequestPast ? "grey.500" : "gray";
+    color = "green";
   } else if (s === HostRequestStatus.HOST_REQUEST_STATUS_REJECTED) {
     icon = <CrossIcon fontSize="inherit" />;
-    color = isRequestPast ? "grey.500" : "red";
+    color = "red";
   } else if (s === HostRequestStatus.HOST_REQUEST_STATUS_PENDING) {
     icon = <QuestionIcon fontSize="inherit" />;
-    color = isRequestPast ? "grey.500" : "gray";
+    color = "gray";
   } else if (s === HostRequestStatus.HOST_REQUEST_STATUS_CANCELLED) {
     icon = <CrossIcon fontSize="inherit" />;
-    color = isRequestPast ? "grey.500" : "gray";
+    color = "red";
```

Comment:
```suggestion
    color = "gray";
```

This one can also be gray since it got cancelled and thus the person didn't stay.